### ISSUE TITLE
New edk2 ovmf downloaded firmware not working

### ISSUE THREAD

**Issue by sawcce** [48cf/limine-zig-template#5]
I wanted to try zig osdev with limine again with the updated template but for some odd reason the updated download link ([Commit with the update](https://github.com/limine-bootloader/limine-zig-template/commit/c3b04de0c1c0909fa4305c45e6cc8fcc6ea59408#diff-e3445fc75aa9c3e4a60fbe5394dcce12693022018216a8fbe0000fe9952850a6R44)) doesn't seem to provide a working firmware?

I tried using the old link and it worked without any issue (although it would seem that it hasn't been updated for a while?).

My current issue is that limine doesn't even boot, it gets stuck on "Guest has not initialized display (yet)". `-d in_asm` does show that the firmware seem to be running.

Software info:
- Ubuntu 22.04.3 LTS on Windows 10 x86_64
- Kernel: 5.15.133.1-microsoft-standard-WSL2

**Comment by sawcce** [comment]
I did check: https://github.com/limine-bootloader/limine-c-template/issues/5
Both links aren't broken for me, it just seems that the nightly builds aren't working as intended.

**Comment by Mintsuki** [comment]
Well I just tried it and it works for me... Maybe it doesn't work on older QEMU?

**Comment by sawcce** [comment]
I'll check my qemu version once I have access to my PC although I did update qemu before cloning the repo so not sure if that's the issue

**Comment by Mintsuki** [comment]
Even then, an updated QEMU on Ubuntu 22.04 is still bound to be an old QEMU...

**Comment by sawcce** [comment]
Yeah I deleted the ovmf folder, ran `make ovmf` then `make run-uefi` and still having the same issue

**Comment by sawcce** [comment]
Even if I just clone the repo without changing anything the same thing happens

**Comment by ethindp** [comment]
I'm suffering something similar: OVMF is able to load, but when it boots from the ISO or HDD (via run-uefi-hdd or run-uefi) it says "booting from HDD"/"booting from DVD" but then just halts. Limine doesn't even show anything. I'm not sure why since I've even enabled `VERBOSE` to see if that fixed it, and it just doesn't print anything on the framebuffer or serial port. The ISO and HDD seem to be set up correctly, so I'm a bit confused.

**Comment by ethindp** [comment]
I found that updating OVMF fixed the bug I mentioned but now Limine is saying that it didn't find any requests, which is weird since I did `pub export var` them.

**Comment by ethindp** [comment]
Okay, getting this bug again. Running on latest Limine. When I boot the vm with make run/run-uefi/run-hdd/run-hdd-uefi`, I get the initial firmware boot messages and such, then a completely blank screen. Running it in GDB shows it's working but I don't know if the debugger is having some kind of influence on the execution of the code or something else.

**Comment by cycl0ne** [comment]
the problem comes that limine wants a really new qemu (not really new, but distro wise), that sometime the distri not have. you need to recompile qemu for your distro as newest or get an old ovmf file from some hobby os here in github. 
@48cf are you still doing something here? otherwise i would like to hijack this project. i started my ZIG quest 2 weeks ago and need a good limine template ;-)
